app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+from typing import Any
+
+
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> Any:
```

Comment:
This method is printing but nothing returns
```suggestion
    def print_name(self) -> None:
```